hex/helpers/build

```diff
@@ -10,7 +10,14 @@ fi
 install_dir="$DEPENDABOT_NATIVE_HELPERS_PATH/hex"
 mkdir -p "$install_dir"
 
-mix local.hex --force
+# Initial Hex install
+mix local.hex --force --if-missing
```

Comment:
`--if-missing` skips installing if already installed... ie, if someone re-running the build script locally for some reason.